README.md

```diff
@@ -64,9 +85,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Microverse
```

Comment:
- [ ] The link to your MIT license is not working as expected, please have another look at [the Microverse template](https://github.com/microverseinc/readme-template) and make sure it works the same.

Reply:
This has been implemented. Thank you